// [[Reason: Function declarations are not fixed]] ////

//// [tests/cases/conformance/declarationEmit/exportDefaultNamespace.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,9 +1,19 @@
 
 
 //// [exportDefaultNamespace.d.ts]
-declare function someFunc(): string;
-declare namespace someFunc {
-    var someProp: string;
-}
-export default someFunc;
-//# sourceMappingURL=exportDefaultNamespace.d.ts.map
\ No newline at end of file
+export default function someFunc(): string;
+//# sourceMappingURL=exportDefaultNamespace.d.ts.map
+/// [Errors] ////
+
+exportDefaultNamespace.ts(1,25): error TS9009: Assigning properties to functions without declaring them is not supported with --isolatedDeclarations. Add an explicit declaration for the properties assigned to this function.
+
+
+==== exportDefaultNamespace.ts (1 errors) ====
+    export default function someFunc(): string {
+                            ~~~~~~~~
+!!! error TS9009: Assigning properties to functions without declaring them is not supported with --isolatedDeclarations. Add an explicit declaration for the properties assigned to this function.
+        return 'hello!';
+    }
+    
+    someFunc.someProp = 'yo';
+    
\ No newline at end of file
